config/config.go

```diff
@@ -21,12 +22,13 @@ type Config struct {
 
 //ServerConfig holds the config for each server the user has setup
 type ServerConfig struct {
-	AccessKey string `json:"accessKey"`
-	SecretKey string `json:"secretKey"`
-	TokenKey  string `json:"tokenKey"`
-	URL       string `json:"url"`
-	Project   string `json:"project"`
-	CACerts   string `json:"cacert"`
+	AccessKey   string                 `json:"accessKey"`
+	SecretKey   string                 `json:"secretKey"`
+	TokenKey    string                 `json:"tokenKey"`
+	URL         string                 `json:"url"`
+	Project     string                 `json:"project"`
+	CACerts     string                 `json:"cacert"`
+	KubeConfigs map[string]*api.Config `json:"kubeConfigs"`
```

Comment:
nit: should this have the YAML fields as well or is the server config usually strictly JSON?

Reply:
This is how it's been, I only added a field. I think most YAML libraries use the json tags now.